test/unit/specs/components/staking/PageValidator.spec.js

```diff
@@ -228,16 +263,174 @@ describe(`PageValidator`, () => {
   })
 })
 
+describe(`modalOptions`, () => {
+  it(`always shows wallet in the first position`, () => {
+    const $store = {
+      commit: jest.fn(),
+      dispatch: jest.fn(),
+      getters: getterValues
+    }
+
+    const {
+      vm: { modalOptions }
+    } = mount(PageValidator, {
+      mocks: {
+        $route: {
+          params: { validator: delegate.owner }
+        },
+        $store
+      }
+    })
+
+    const myWallet = {
+      address: $store.getters.wallet.address,
+      maximum: Math.floor(
+        $store.getters.totalAtoms - $store.getters.oldBondedAtoms
+      ),
+      key: `My Wallet - ${shortAddress($store.getters.wallet.address, 20)}`,
+      value: 0
+    }
+
+    let options = modalOptions()
+    expect(options).toHaveLength(1)
+    expect(options[0]).toEqual(myWallet)
+  })
+
+  it(`only shows wallet if the only bonded validator is the same displayed on page`, () => {
```

Comment:
Why is that?

Reply:
We discussed  this week that we won't give the option to redelegate to the same bonded validator

Reply:
Changing it to `hides displayed validator if bonded`. I think is more descriptive